app/views/devise/registrations/edit.html.erb

```diff
@@ -1,42 +1,42 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
+<h1 class="h4 text-gray-900 mb-4">Edit <%= resource_name.to_s.humanize %></h1>
 
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
+  <div class="form-group">
     <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "Email" %>
+    <%= f.email_field :email, autofocus: true, autocomplete: "Email", class: "form-control", required: true  %>
   </div>
 
   <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
     <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
   <% end %>
 
-  <div class="field">
+  <div class="form-group">
     <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
-    <%= f.password_field :password, autocomplete: "New Password" %>
+    <%= f.password_field :password, autocomplete: "New Password", class: "form-control", required: true  %>
     <% if @minimum_password_length %>
       <br />
       <em><%= @minimum_password_length %> characters minimum</em>
     <% end %>
   </div>
 
-  <div class="field">
+  <div class="form-group">
     <%= f.label :password_confirmation %><br />
-    <%= f.password_field :password_confirmation, autocomplete: "New Password" %>
+    <%= f.password_field :password_confirmation, autocomplete: "New Password", class: "form-control", required: true %>
   </div>
 
-  <div class="field">
+  <div class="form-group">
     <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
```

Comment:
Internationalization. Please verify throughout the code

Reply:
done